CHANGELOG

```diff
@@ -1,9 +1,53 @@
 Version 2024.03 (unreleased)
   Friendica Core
+    Updates to the translations AR, BG, CS, DE, EO, ES, FR, GD, HU, IS, IT, JA, PL, RO, RU, SV
+    Updates to the themes (frio, vier) [annando, foss-, haheute, MrPetovan, Raroun, toddy15]
+    Improved the channel feature [annando]
+    Improved the search performance [annando]
+    Improved spam detection [annando]
+    Improved the account overview on the moderation page [annando]
+    Improved account creation via CLI console [mexon]
+    Improved the Mastodon compatible API [annando, MrPetovan]
+    Improved logging of the system load value [annando]
+    Improved image handling [annando]
+    Fixed an issue that could lead to empty URLs in the server block list [annando]
+    Fixed XSS attacks [leoOliver, MrPetovan, snajafov]
+    Fixed enumberation attack on photo URLs [MrPetovan]
```

Comment:
This has been postponed to a future release.

Reply:
Ok